src/app/components/Speakers.js

```diff
@@ -0,0 +1,17 @@
+import React, { Component } from 'react'
+import config from '../config.json'
+
+class Speakers extends Component {
+  shouldRender (edition) {
+    return config[edition].indexOf('Speakers') > -1
+  }
+  render () {
+    return (
+      <div>
+        { this.shouldRender(this.props.edition) && <h3> The speakers component is rendered</h3> }
```

Comment:
can't this evaluating be done on the parent component. This component will always render, you are just specifying if it will be empty or not, right?